agent/provider/src/cli/rule.rs

```diff
@@ -147,7 +159,11 @@ impl From<RulesManager> for RulesTable {
 
         table.with_header(outbound.enabled);
         table.add_everyone(outbound.everyone);
-        table.add(outbound.audited_payload.default);
+        table.add_audited_payload(outbound.audited_payload.default);
+
+        for (cert_id, rule) in outbound.partner.iter() {
+            table.add_partner(cert_id, rule);
+        }
```

Comment:
I'd use the same convention as between lines 160-162 -> 
`table.add_partner(outbound.partner);`

and the iterate by (cert_id, rule) under `add_partner` which should be sth like

```
    fn add_partner(&mut self, partner: HashMap<String, CertRule>) {
        let _ = partner.iter().map(|(cert_id, rule)| {
            let row = serde_json::json! {[ "Partner", rule.mode, cert_id, rule.description ]};

            self.table.values.push(row);
        });
    }